### DISCUSSION TITLE
Run RuinedFooocus on Colab

### DISCUSSION THREAD

**Discussion by rajtilakjee** [runew0lf/RuinedFooocus#126] [Show and tell]
Go to the following link and click on the Open in Colab button: https://github.com/rajtilakjee/ruinedfooocus-on-cloud

**Comment by arlojeremy** [comment]
What did you do to make this in colab??

**Comment by arlojeremy** [comment]
What did you do to turn it into a colab? I'm getting a 404.

**Reply by rajtilakjee** [reply]
This repo is not being maintained for the time being. Feel free to fork it and see if you can solve the issues.

**Comment by yownas** [comment]
I played around a little and think I got things working. It starts but there are probably a bunch of things to fix, like making it save images in a better place.

https://gist.github.com/yownas/89de44e0f0d057e06e2a2dcf4d5b392c
